crates/red_knot_python_semantic/resources/mdtest/generics.md

```diff
@@ -0,0 +1,41 @@
+# PEP 695 Generics
+
+## Class Declarations
+
+Basic PEP 695 generics
+
+```py
+class MyBox[T]:
+  data: T
+  box_model_number = 695
+  def __init__(self, data: T):
+    self.data = data
+
+# TODO not error (should be subscriptable)
+box: MyBox[int] = MyBox(5)  # error: [non-subscriptable]
+# TODO error differently (str and int don't unify)
+wrong_innards: MyBox[int] = MyBox("five")  # error: [non-subscriptable]
+# TODO reveal int
+reveal_type(box.data)  # revealed: @Todo
+
+reveal_type(MyBox.box_model_number)  # revealed: Literal[695]
+```
+
+## Subclassing
```

Comment:
I would like to also add a test that in a stub file (`pyi`) we can have a generic class that refers to itself in its own bases. We have a similar test already for non-generic classes.

Reply:
Added